index.html

```diff
@@ -9,261 +9,392 @@
-                <h3>Get started</h3>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
-            </div>
-            <form action="https://formspree.io/f/xbjwgkqr" method="post">
-                <input type="text" name="name" id="name" placeholder="Your name" maxlength="30" required>
-                <input type="email" name="email" id="email" placeholder="Your email" required>
-                
-                <textarea name="message" id="msg" cols="30" rows="10" placeholder="Enter your message here..." maxlength="500" required></textarea>
-                <button class="btn-collabo" type="submit" name="submit">Start collaboration</button>
-            </form>
-        </div>
-        <div class="social-footer">
-            <div class="social-container">
-                <a href="#">
-                    <p>Get my Resume <span><img src="images/resume.png" alt="image"/></span></p>
-                </a>
-                <ul>
-                    <li><a href="#"><img src="images/stackoverflow.png" class="fa-img" alt="image"></a></li>
-                    <li><a href="#"><img src="images/github.png" class="fa-img" alt="image"></a></li>
-                    <li><a href="#"><img src="images/twitter.png" class="fa-img" alt="image"></a></li>
-                    <li><a href="#"><img src="images/linkedin.png" class="fa-img" alt="image"></a></li>
-                    <li><a href="#"><img src="images/m-vector.png" class="fa-img" alt="image"></a></li>
-                    <li><a href="#"><img src="images/fa-vector.png" class="fa-img" alt="image"></a></li>
-                 </ul>
-                
+                <div class="img-cont-dsk-right display-none">
+                    <img src="images/image-geometry_7.png" alt="image">
+                </div>
+
             </div>
-        </div>
-    </footer>
+             <div class="social-footer">
+                 <div class="social-container">
+                     <a href="#">
+                         <p>Get my Resume <span><img src="images/resume.png" alt="image"/></span></p>
+                     </a>
+                     <ul>
+                         <li><a href="#"><img src="images/stackoverflow.png" class="fa-img" alt="image"></a></li>
+                         <li><a href="#"><img src="images/github.png" class="fa-img" alt="image"></a></li>
+                         <li><a href="#"><img src="images/twitter.png" class="fa-img" alt="image"></a></li>
+                         <li><a href="#"><img src="images/linkedin.png" class="fa-img" alt="image"></a></li>
+                         <li><a href="#"><img src="images/m-vector.png" class="fa-img" alt="image"></a></li>
+                         <li><a href="#"><img src="images/fa-vector.png" class="fa-img" alt="image"></a></li>
+                      </ul>        
+                 </div>
+             </div>
+         </footer>      
+    </div>
```

Comment:
- [ ] Optional: I recommend to delete extra white space below the footer in mobile view: 
<img width="321" alt="Screenshot 2022-02-25 at 13 10 02" src="https://user-images.githubusercontent.com/77038610/155687637-4fb62d09-308a-4af2-a405-63258fb0b914.png">